storage/storage.go

```diff
@@ -67,12 +68,18 @@ func NewClient(ctx context.Context, url *url.URL, opts Options) (Storage, error)
 
 // Options stores configuration for storage.
 type Options struct {
-	MaxRetries    int
-	Endpoint      string
-	NoVerifySSL   bool
-	DryRun        bool
+	MaxRetries  int
+	Endpoint    string
+	NoVerifySSL bool
+	DryRun      bool
 	NoSignRequest bool
-	bucket        string
+	bucket      string
+	region      string
+}
+
+func (o Options) SetRegion(region string) Options {
+	o.region = region
```

Comment:
SetRegion won't work because receiver is not a pointer. https://play.golang.org/p/y_6xsO3zB9L

Reply:
Actually it works here since I'm using `opt = opt.SetRegion(...)`. Anyway, I will change this to make it consistent with other setters.